src/repl.ml

```diff
@@ -199,4 +251,22 @@ let register extension instance =
           Extension_instance.close_repl instance)
       ()
   in
+  Vscode.ExtensionContext.subscribe extension ~disposable;
+  let disposable =
+    Vscode.Workspace.onDidChangeTextDocument () ~listener:(fun event ->
```

Comment:
I added this `onDidChangeTextDocument` that unsets the highlighted color if there is any update inside or before the evaluated region.